docs/running-as-firefox-panel.md

```diff
@@ -0,0 +1,98 @@
+# Running as the firefox panel
+
+* [Getting Started with Firefox Nightly](#getting-started-with-firefox-nightly)
+  * [For Windows Developers](#for-windows-developers)
+* [Configuring Firefox path in the debugger](#configuring-firefox-path-in-the-debugger)
+* [Watching for Changes](#watching-for-changes)
+* [Running Nightly with the local debugger](#running-nightly-with-the-local-debugger)
+* [Getting Help](#getting-help)
+
+## Getting Started with Firefox Nightly
+
+Running firefox is similar to running [Mochitests](./mochitests.md)
+
+**Requirements**
+
+* mercurial ( `brew install mercurial` )
+* autoconf213 ( `brew install autoconf@2.13 && brew unlink autoconf` )
+
+On the first run, this will download a local copy of Firefox and set up an [artifact build](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Artifact_builds) (just think of a super fast Firefox build). It may take a while (10-15 minutes) to download and build Firefox.
+
+There are two commands that you will need. `./mach build` and `./mach run` these can be run together
+each time things change with `./mach build && ./mach run`
+
+* The first time you run the project you will need to use `./mach build` but after the first run you can use `./mach build faster`! The full command looks like this`./mach build faster && ./mach run`
```

Comment:
lets link to docs: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/mach
I'd rather people learn about edge cases there!